src/main/daml/Daml/Finance/Claims/Util.daml

```diff
@@ -24,12 +24,8 @@ isZero' = all $ CC.isZero . (.claim)
 
 -- | Maps the time parameter in a `Claim` to `Time`. As `Time` is generally understood to express UTC time, we recommend mapping to UTC time.
 toTime : HasUTCTimeConversion t => Claim t x a o -> Claim Time x a o
-toTime =
-  let contraMap _ = error "Inverse mapping from `Time` is not provided" -- currently the contramap from `Time` back to `a` is not used, because `Observation`\s do not depend on time explicitly
```

Comment:
This comment kind of worries me, because `Observable`s  _are_ time dependent, and are supposed to represent some kind of `t -> *` to put it loosely. Even if the parameter is not explicit.

Reply:
They are indeed time-dependent, it's just that time is never used in the actual data type (unlike with `Inequalities`). We just call the observable e.g.`Observe SPOT AAPL` with implicit time-dependence.

It would be different if we had time-dependent observations such as `t - T`. I wonder if we could use time-dependent deterministic observations to take into account stuff like day-count fractions, that could probably work.